doc/changelog/09-coqide/11415-remove-ide-revert-all-buffers.rst

```diff
@@ -0,0 +1,4 @@
+- **Removed:**
+  Removed the "Revert all buffers" command from CoqIDE which had been broken for a long time
```

Comment:
Following Hugo's comment, here's an alternative suggestion:

```suggestion
  Removed the "Revert all buffers" command from CoqIDE whose purpose was counter-intuitive / unclear to users.
```

Of course, we can also keep the current wording as it all depends on what meaning we give to the word "broken".